lib/linguist/languages.yml

```diff
@@ -2668,6 +2669,16 @@ OpenSCAD:
   tm_scope: none
   ace_mode: scad
 
+OpenType Font Feature:
+  type: programming
```

Comment:
The hell. I didn't even notice PostScript was classed as markup, and I was even editing that part the other night.

Right, time to amend.